### PR TITLE
Create Support File

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary

Added markdown file for support, GitHub will know about the file and put it on the sidebar under `Helpful resources` over there ----->

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [X] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by RDIL** [comment]
Great, failing build!!

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# PowerShell Support  
+If you have any problems, please consult the [known issues][], developer [FAQ][], and [GitHub issues][].  
+If you do not see your problem captured, please file a [new issue][] and follow the provided template.  
+
+[FAQ]: https://github.com/PowerShell/PowerShell/tree/master/docs/FAQ.md
```
Comment (TravisEz13) [inline .github/SUPPORT.md]: Add a to the PowerShell Core life cycle which include the official support policy:
https://docs.microsoft.com/en-us/powershell/scripting/powershell-support-lifecycle?view=powershell-6

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comment

File: .github/SUPPORT.md (hunk 1 above)
Comment (RDIL) [reply .github/SUPPORT.md]: 👍

**Comment by RDIL** [comment]
fixed!

**Comment by RDIL** [comment]
@TravisEz13 Fixed the feedback, also WHY CODACITY CodeFactor is so much better!!

**Comment by RDIL** [comment]
Codacity is gone yay

**Comment by TravisEz13** [comment]
Codacity Supports PowerShell Script analyzer.  We can consider disabling rules.  It also run analysis we are required to look at and evaluate the results.

**Comment by TravisEz13** [comment]
I rebased your PR to master to trigger the latest static analysis.

**Comment by RDIL** [comment]
*failed* *failed again*

**Comment by RDIL** [comment]
What does this mean?:
```
2019-01-16T23:19:44.0996261Z   Describing Common Tests - Validate Markdown Files
2019-01-16T23:19:44.0996343Z VERBOSE: NPM is checking Gulp is installed. This may take a few moments.
2019-01-16T23:20:08.6540685Z     [+] Should not have errors in any markdown files 25.25s
2019-01-16T23:20:08.6717885Z Tests completed in 159.46s
2019-01-16T23:20:08.6972505Z Tests Passed: 431, Failed: 1, Skipped: 0, Pending: 2, Inconclusive: 0 
2019-01-16T23:20:09.9762947Z markdown link tests failed
2019-01-16T23:20:09.9763679Z At /home/vsts/work/_temp/556706ef-519c-4971-8ccf-5d3722c2cbeb.ps1:8 char:3
2019-01-16T23:20:09.9763748Z +   throw "markdown link tests failed"
2019-01-16T23:20:09.9763840Z +   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2019-01-16T23:20:09.9763892Z + CategoryInfo          : OperationStopped: (markdown link tests failed:String) [], RuntimeException
2019-01-16T23:20:09.9763971Z + FullyQualifiedErrorId : markdown link tests failed
2019-01-16T23:20:09.9764053Z  
2019-01-16T23:20:09.9817530Z ##[error]PowerShell exited with code '1'.
```

**Comment by TravisEz13** [comment]
I'll fix the link issue.

**Comment by TravisEz13** [comment]
Go to the tests tab and it will show you what test failed.

**Comment by RDIL** [comment]
Oh that exists thanks

**Comment by TravisEz13** [comment]
The windows failure is a very intermittent failure.  It should pass on rerun.  I'm not going to rerun because we should make the broken link relative so the PR will pass.

**Comment by RDIL** [comment]
Thanks.

**Comment by TravisEz13** [comment]
Actually, I was thinking about it and the link cannot be relative.  I'll force merge with the failure.  I've reset back to before I made the change.

**Comment by RDIL** [comment]
Oh ok

**Comment by TravisEz13** [comment]
Used admin to merge.  The only failure was a new link added in this PR which failed to verify.

@RDIL 	We could adjust the test for this scenario so that if the URL has `?AddedInPr=8618` where the PR matches the current PR number, it ignores the failure.  Most sites will ignore the query string for static content.

@adityapatwardhan 	What do you think?

**Pull request merged**